### DISCUSSION TITLE
Admin settings Users: cannot change "allow chat deletion" option

### DISCUSSION THREAD

**Discussion by Hugobox** [open-webui/open-webui#3374] [General]
# Bug Report

## Description

**Bug Summary:**
When clicking on the Admin Settings Users tab and trying to change the "allow chat deletion", the chrome debug shows an error and the openwebui webapp freezes until an F5 refresh.

**Steps to Reproduce:**
Go to Admin Settings Users tab and trying to change the "allow chat deletion" while the F12 chrome console is opened
Noticed the errror

**Expected Behavior:**
Should be able to toggle the option on of off
**Actual Behavior:**
Toggle stays on
## Environment

- **Open WebUI Version:** 0.3.5

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**

Users.svelte:61 Uncaught 
TypeError: Cannot set properties of undefined (setting 'deletion')
    at HTMLButtonElement.p (Users.svelte:61:24)

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
Can't reproduce the issue :/ let us know if you figure out what the culprit was!
